build/ci/community/wait_demo_pr_checks.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash -xe
+set +o pipefail
+
+wait_fot_checks_to_start(){
+  community_operators_branch=$1
+  forked_community_operators_repository=$2
+  gh_pr_checks_command="gh pr checks $community_operators_branch --repo $forked_community_operators_repository"
+  while [ `$gh_pr_checks_command | grep -i pending | wc -l` -eq 0 ]; do
+    sleep 1
+  done
+}
+wait_for_checks_to_complete(){
+  community_operators_branch=$1
+  all_tests_passed=false
+  gh_pr_checks_command="gh pr checks $community_operators_branch --repo $forked_community_operators_repository"
```

Comment:
can `gh_pr_checks_command` be defined once for both functions?

Reply:
done